node/child_process_test.ts

```diff
@@ -487,3 +487,22 @@ Deno.test({
     assertExists(childProcess.exitCode);
   },
 });
+
+Deno.test({
+  name: "[node/child_process] ChildProcess.unref()",
+  async fn() {
+    const script = path.join(
+      path.dirname(path.fromFileUrl(import.meta.url)),
+      "./testdata/child_process_unref.js",
+    );
+    const childProcess = spawn(Deno.execPath(), [
+      "run",
+      "-A",
+      "--unstable",
+      script,
+    ]);
+    const p = deferred();
+    childProcess.on("exit", () => p.resolve());
+    await p;
```

Comment:
Will the test runner `await` `p` for you if you return it here instead of awaiting it manually? It's not a big deal either way.

Reply:
I don't think so